.github/workflows/test.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           cd /tmp
           git clone --branch ${{ github.event.inputs.branch }} --depth 1 https://github.com/percy/cli
           cd cli
-          PERCY_PACKAGES=`find packages -type d -depth 1 | sed -e 's/packages/@percy/g' | tr '\n' ' '`
+          PERCY_PACKAGES=`find packages -mindepth 1 -maxdepth 1 -type d | sed -e 's/packages/@percy/g' | tr '\n' ' '`
```

Comment:
* What are the inconsistencies, we're using `ubuntu-latest` everywhere right, why will it not cause issues in other places and just here?
* If we make this change, we'll have to make these changes consistent across SDKs.

Reply:
Not sure why it was working in others. It didn't error out in any other SDK. We'll have to cross check those as well.

Reply:
Turns out it wasn't working in others properly as well. Raised PRs to fix those